setup.py

```diff
@@ -103,6 +103,7 @@
     "deepspeed": ["deepspeed>=0.14.4"],
     "quantization": ["bitsandbytes<=0.41.1"],
     "llm_judge": ["openai>=1.23.2", "llm-blender>=0.0.2"],
+    "mergekit": ["mergekit @ git+https://github.com/arcee-ai/mergekit.git"],
```

Comment:
This is not ideal and very messy, but it was the only way I could install mergekit from source since "pip" installed an older version

Reply:
The `mergekit` maintainer has kindly made a new release. Would you mind setting that with a lower bound and seeing if the callback still works?

Reply:
It works the same overall, but `mergekit` 0.0.5 uses an older version of `accelerate`, which breaks training unless you upgrade `accelerate` afterward. installing `mergekit` from source doesn't cause this problem

Reply:
Sorry about that! 0.0.5.1 targets the current version of `accelerate`.